biztalk/core/office365-mail-adapter.md

```diff
@@ -99,6 +99,26 @@ You can use a simple File receive port and location to send messages to your Off
 4. Start the File receive location and the Office 365 Outlook Email adapter send port.
 5. Copy **Office365Mail.xml** sample message into the receive folder (C:\\Temp\\In\\). The send port sends the XML file as mail body to your Office 365 Outlook Email. Files specified in the Attachments tab of the port configuration are attached to the email.
 
+### Additional configuration in the send handler
+
+By default, the Office 365 Outlook Email adapter sends 20 emails at a time across all its send ports in the host instance. You can adjust this number by setting the **Maximum concurrent sends** property in the Send handler properties for the Office 365 Outlook Email adapter. This property can be used for instance to fine-tune the message throughput when the email service provider limits the number of messages that can be sent over a period of time. Use the following procedure to change the value:

```

Comment:
This will not hold true for customers on older versions (2020 RTM or a previous CU). I think it would be good to mention that.